tests/crafting_test.cpp

```diff
@@ -910,6 +910,9 @@ TEST_CASE( "tools_use_charge_to_craft", "[crafting][charge]" )
             item soldering_iron_portable( "soldering_iron_portable" );
             soldering_iron_portable.put_in( item( "battery_ups" ), pocket_type::MOD );
             tools.push_back( soldering_iron_portable );
+            item plastic_molding = item( "vac_mold" );
+            plastic_molding.put_in( item( "battery_ups" ), pocket_type::MOD );
+            tools.push_back( plastic_molding );
```

Comment:
We need to explicitly add and UPS power the vacuum molder here, otherwise the test assertion will give a false positive, since item would not be craftable without vacuum molder being powered up.